types/result.go

```diff
@@ -249,6 +250,27 @@ func (r TxResponse) Empty() bool {
 	return r.TxHash == "" && r.Logs == nil
 }
 
+// SearchTxsResult defines a structure for querying txs pageable
+type SearchTxsResult struct {
+	TotalCount int          `json:"total_count"` // Count of all txs
```

Comment:
Does this need to returned with every transaction search query? Should this be it's own query?

Reply:
I think any query which does not return all items one time should be pageable. This is friendly to the user.

Reply:
I think any query which does not return all items one time should be pageable. This is friendly to the user. @jackzampolin